src/frontend/styles/components/sidebar.pcss

```diff
@@ -1,6 +1,34 @@
 .docs-sidebar {
   width: 100vw;
-    
+
+  &--animated {
+    .docs-sidebar__content {
+      transition: transform 200ms ease-in-out;
```

Comment:
Please, add the "will-change: transform" to the `.docs-sidebar__content`'s default rules set to optimize the repaint

Reply:
Fixed